.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -23,6 +23,7 @@ assignees: ''
  - [ ] [COM] inform `#release_coordination` and `#marketing` in internal chat
  - [ ] [GIT] Merge branch `release_Y.Y.Y` in owncloud/android-library, into master
  - [ ] [GIT] Merge branch `release_X.X.X` in owncloud/android, into master
+ - [ ] [DIS] Upload release APK to internal owncloud instance
```

Comment:
I would upload the release apk to internal after uploading it to Play Store since the console itself performs extra checks over the apk. It's not usual to find errors at this point but it could be possible, e.g. versionName or versionCode lower than the apk already published in PlayStore or others.

Reply:
ok, will re-allocate it